static/app/components/events/interfaces/nativeFrame.tsx

```diff
@@ -346,8 +340,10 @@ const AddressCell = styled('div')`
   ${p => p.onClick && `color:` + p.theme.linkColor};
 `;
 
-const FunctionNameCell = styled('div')`
+const FunctionNameCell = styled('div')<{isSystemLabel: boolean}>`
   word-break: break-all;
+  color: ${p => (p.isSystemLabel ? p.theme.subText : 'default')};
```

Comment:
Is `default` a valid color/font-style? I think you want to only apply these styles conditionally, like:

```ts
${p => p.isSystemLabel
 ?  css`
      color: ${p.theme.subText};
	  font-style: 'italic';
     ` 
  : ''
```